src/test/java/sg/dex/starfish/impl/TestResources.java

```diff
@@ -15,18 +17,12 @@
 public class TestResources {
 
     @Test
-    public void testResourceAssetWithContentHash() {
+    public void testResourceAssetWithContentHash() throws DigestException {
         DataAsset dataAsset = ResourceAsset.create("assets/hello.txt");
         byte[] bs = dataAsset.getContent();
-        String s = new String(bs, StandardCharsets.UTF_8);
-        Assertions.assertEquals("Hello Starfish", s);
-
-        Assertions.assertNotNull(dataAsset.getMetadataString());
-        Assertions.assertEquals(14, dataAsset.getContentSize());
-
         assertFalse(dataAsset.getMetadata().containsKey(Constant.CONTENT_HASH));
         dataAsset = dataAsset.includeContentHash();
-        Assertions.assertEquals("e4be09f07f5665ecacc078223e86c1dba18b38a3e07a3d575167b5ba7a1821d1", dataAsset.getMetadata().get(Constant.CONTENT_HASH));
```

Comment:
Please test for the the correct hash value!

Reply:
Fixed

Reply:
Should test for a specific hex string - why are we removing this line?

Reply:
this value was not clear to user , so I calculated the hex value using SHA3 and used the same for validation.Variable expected is the same value as e4be09f07f5665ecacc078223e86c1dba18b38a3e07a3d575167b5ba7a1821d1.